bootstrap.sh

```diff
@@ -1,26 +1,61 @@
 #!/bin/sh
 
+function pause {
+ 	read -s -n 1 -p "Press any key to continue . . ."
+ 	echo ""
+}
+
+function success {
+	echo Bootstrap succeeded \!
+	exit 0
+}
+
+function error {
+	echo Bootstrap failed \!
+	pause
+	exit 1
+}
+
 # fail immediately if anything goes wrong
 set -e
 
+pushd $(dirname $0) > /dev/null
+CURRENT_DIR=$(pwd)
+popd > /dev/null
+
+which msbuild > /dev/null
+MSBUILD_FOUND=$?
+if [ $MSBUILD_FOUND -ne 0 ]; then
+    echo "MSBuild not found"
+    error
+fi
+
 # workaround for https://github.com/mono/mono/issues/6752
 TERM=xterm
 
 # TODO: Test mono and msbuild existence
```

Comment:
since you've tested for their existence, remove the TODO (and thanks! ;) )

Reply:
I'll do it ;)

Reply:
Did in last commit "4bb56e39b5db34227fa62dfc54c4b83b37ec2563"